android/src/main/java/com/dooboolab/fluttersound/FlutterSoundPlugin.java

```diff
@@ -427,37 +475,29 @@ public void run() {
     }
   }
 
-  @Override
-  public void startPlayer(final String path, final Result result)
+  public void startPlayerFromBuffer(final byte[] dataBuffer, t_CODEC codec, final Result result)
   {
-    _startPlayer(path, null, result) ;
-  }
+    String extentionArray[] = {
+              "acc"   // DEFAULT
+            , "acc"   // CODEC_AAC
+            , "opus"  // CODEC_OPUS
+            , "caf"   // CODEC_CAF_OPUS (this is apple specific)
+            , "mp3"   // CODEC_MP3
+            , "ogg"   // CODEC_VORBIS
+            , "wav"   // CODEC_PCM
+    };
```

Comment:
Same here!

Reply:
You are perfectly right. I will but those constants array outside the procedure.